### DISCUSSION TITLE
Cannot download "enwiki_core" in spikeX

### DISCUSSION THREAD

**Discussion by Hannah123567** [explosion/spaCy#11821] [Help: Installation] [answered]
The spikex provides wikigraphs ready to be used, including enwiki_core, simplewiki_core, and itwiki_core.
However, it seems that the website that stores these packages has gone (404 not found) so that we can not download  "enwiki_core". How can I access to the wikigraph package (i.e., enwiki_core) at this moment? 
I would be very grateful if someone could help me download these packages.

**Comment by ljvmiranda921** [comment, marked as answer]
Hi @Hannah123567 , the SpikeX package is a third-party module from the [spaCy Universe](https://spacy.io/universe/project/spikex), you might be better off contacting the [authors directly](https://github.com/erre-quadro/spikex)  (I see you've already done that) as we don't have control of SpikeX's download servers.

**Reply by Hannah123567** [reply]
Hi @ljvmiranda921,  thank you very much for your prompt reply!  Indeed I am looking for packages that can do sentence-level topic modeling. I would be very grateful if you have recommendations on other packages in Spacy Universe.

**Reply by ljvmiranda921** [reply]
You can find all [spaCy universe](https://spacy.io/universe) packages here. One that might be relevant to your use-case is [tmtoolkit](https://spacy.io/universe/project/tmtoolkit).
